Import `field-sizing` WPT tests

lukewarlow opened the pull request:
#### 24a12aa6d3bbf28e24c8c7dbe186377e5e3c432e
<pre>
Import `field-sizing` WPT tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=268954">https://bugs.webkit.org/show_bug.cgi?id=268954</a>

Reviewed by Aditya Keerthi.

Imports CSS parsing and HTML rendering tests for the field-sizing CSS property

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/3c00d743109ae3d06e874e56dd9287f3ded06f44">https://github.com/web-platform-tests/wpt/commit/3c00d743109ae3d06e874e56dd9287f3ded06f44</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/css/css-ui/parsing/field-sizing-computed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ui/parsing/field-sizing-computed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ui/parsing/field-sizing-invalid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ui/parsing/field-sizing-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ui/parsing/field-sizing-valid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ui/parsing/field-sizing-valid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ui/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-number-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-number.tentative-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-number.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-number.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-text-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-text.tentative-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-text.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-text.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-select-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-select.tentative-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-select.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-select.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-textarea-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-textarea.tentative-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-textarea.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-textarea.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/w3c-import.log:
* LayoutTests/platform/glib/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/274299@main">https://commits.webkit.org/274299@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e4069531bc441868c86f429e25b3627a50d35fcb

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/38566 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/17497 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/40892 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/41101 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/34237 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/20292 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/14840 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/32417 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/39139 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/14750 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/33550 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/12807 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/12793 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/34411 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/42377 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/35036 "Passed tests") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/34822 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/38619 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/13400 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/11080 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/36827 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/15006 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/33710 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/13866 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/5033 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/14480 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

pxlcoder reviewed (approved):
LGTM, let's make sure EWS is passing everywhere before merging

lukewarlow committed:
Import `field-sizing` WPT tests
https://bugs.webkit.org/show_bug.cgi?id=268954

Reviewed by Aditya Keerthi.

Imports CSS parsing and HTML rendering tests for the field-sizing CSS property

Upstream commit: https://github.com/web-platform-tests/wpt/commit/3c00d743109ae3d06e874e56dd9287f3ded06f44

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/css/css-ui/parsing/field-sizing-computed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ui/parsing/field-sizing-computed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ui/parsing/field-sizing-invalid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ui/parsing/field-sizing-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ui/parsing/field-sizing-valid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ui/parsing/field-sizing-valid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-ui/parsing/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-number-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-number.tentative-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-number.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-number.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-text-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-text.tentative-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-text.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-text.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-select-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-select.tentative-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-select.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-select.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-textarea-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-textarea.tentative-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-textarea.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-textarea.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/html/rendering/widgets/w3c-import.log:
* LayoutTests/platform/glib/TestExpectations:

Canonical link: https://commits.webkit.org/274299@main
